### DISCUSSION TITLE
v-bind and client:only conflicted

### DISCUSSION THREAD

**Discussion by joesonw** [ElMassimo/iles#257] [General]
I am using both `v-bind` and `client:load` script on a component, it will produce `Duplicate attribute` error.

I suspect it's here
https://github.com/ElMassimo/iles/blob/86ea918eef71a4ac3ffe212cae1b22fde28e1554/packages/iles/src/node/plugin/wrap.ts#L150-L158

is there any way around it?

**Comment by ElMassimo** [comment]
Hi Joeson!

Are you passing any props to the island component? Trying to understand if the _Duplicate attribute_ error is caused by a difference in casing (otherwise merging `props` with the other properties would remove duplicates).

If you provide a small reproduction I can take a look.

**Reply by joesonw** [reply]
Yea.
So basically, I am using `client:load` script. And I also used `v-bind` on the component. I guess that's where it conflicted.
